funnel/views/profile.py

```diff
@@ -242,6 +243,61 @@ def view(self) -> ReturnRenderWith:
 
         return ctx
 
+    def jsonify_calendar(jsonData):
+        return jsonify(jsonData['projects'])
+
+    @route('calendar', methods=['GET'], endpoint='calendar')
+    @requestargs('start', 'end')
+    @render_with(
+        {
+            'text/html': 'profile_calendar.html.jinja2',
+            'application/json': jsonify_calendar,
+        }
+    )
+    def org_calendar(
+        self, start: datetime = None, end: datetime = None
+    ) -> ReturnRenderWith:
+        projects = []
+        if start is not None and end is not None:
+            start = datetime.strptime(start, "%Y-%m-%dT%H:%M:%S%z")
+            end = datetime.strptime(end, "%Y-%m-%dT%H:%M:%S%z")
+            all_projects = self.obj.listed_projects.order_by(None)
+            if end > start:
+                filtered_projects = (
+                    all_projects.filter(
+                        Project.start_at >= start,
+                        Project.end_at < end,
+                    )
+                    .order_by(Project.order_by_date())
+                    .all()
+                )
+                projects = [
+                    {
+                        'title': p.title,
+                        'start': p.start_at,
+                        'end': p.end_at,
+                        'date_str': datetime_filter(
+                            p.start_at_localized, format='dd MMM yyyy'
+                        ),
+                        'time': datetime_filter(p.start_at_localized, format='hh:mm a'),
+                        'venue': (
+                            p.primary_venue.city if p.primary_venue else p.location
+                        ),
+                        'cfp_open': True if p.cfp_state.OPEN else False,
+                        'member_access': (
+                            True
+                            if (p.features.rsvp_for_members or p.features.subscription)
+                            else False
+                        ),
+                        'url': p.url_for(),
+                    }
+                    for p in filtered_projects
+                ]
+        return {
+            'profile': self.obj.current_access(datasets=('primary', 'related')),
```

Comment:
If the model declares `__json_datasets__`, that is used automatically, so the `datasets` parameter can be removed here.